cset-workflow/meta/rose-meta.conf

```diff
@@ -676,6 +676,107 @@ type=python_boolean
 compulsory=true
 sort-key=stash2
 
+[template variables=BASIC_QQ_PLOT]
+ns=Diagnostics
+description=Create a basic quantile-quantile plot for variables specified collapsing over specified coordinates.
+help=The quantiles defined for each variable are 0, 0.001, 0.01,
+     0.1,1,5,10,15,20,25,30,40,50,60,70,75,80,85,90,95,99,99.9,99.99,
+     99.999,100.
+     For further help see includes/basic_qq_plot.cylc
+trigger=template variables=MODEL_FIELDS_A: True;
+        template variables=MODEL_FIELDS_B: True;
+        template variables=COORDINATE_LIST: True;
+        template variables=ONE_TO_ONE: True;
+type=python_boolean
+compulsory=true
+sort-key=qq1
+
+[template variables=MODEL_FIELDS_A]
+ns=Diagnostics
+description=List of standard names for model fields to plot on y axis.
+help=Include a list of variable names in python list format["var1","var2"].
```

Comment:
It will also fail if "air_temperature" or "relative_humidity" are used and pressure level data as well as surface variable data is around, as the filters will return too many cubes

Reply:
updated to have a levels constraint as well, applied for A and B so that 200 hPa of model field A could be compared against a different height for model field B if required.